.gitignore

```diff
@@ -47,7 +47,10 @@ coverage.xml
 .hypothesis/
 .kitchen
 .kitchen.local.yml
+<<<<<<< HEAD
```

Comment:
some rebase stuff has been left behind..

Reply:
c**p, thanks @aboe76 . I'll do some greps to check and fix that.